### ISSUE TITLE
 error in isotope-guided-walk.R: the object d18o_world on line 123 wasn't found

### ISSUE THREAD

**Issue by adw96** [statdivlab/KikWalk#1]
Excellent question received by email: running `isotope-guided-walk.R` leads to an error on line 123:

```
Error in calRaster(known = d, isoscape = d18o_world) : 
  object 'd18o_world' not found
```

Update pending and commit to follow

**Comment by adw96** [comment]
It appears that this issue arose when the authors of the `assignR` package altered `assignR::calRaster` -- some time between v2.1.1 and 1.2.1 (we developed this software under the latter release). One can check the `assignR` version using `sessionInfo()`. Running the following code appears to fix the error:

```
devtools::install_version("assignR", version = "1.2.1", repos = "http://cran.us.r-project.org")
library(assignR) 
sessionInfo() # confirm assignR_1.2.1
data(d18o_world)
```

**Comment by adw96** [comment]
Issue fixed in commit `c189f70`
